packages/commonwealth/client/scripts/views/pages/RewardsPage/tables/WalletTable/WalletTable.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 
 import './WalletTable.scss';
+import XPTable from './XPTable';
 
 export const WalletTable = () => {
-  return <div className="WalletTable">Wallet Table</div>;
+  return (
+    <div className="WalletTable">
+      <XPTable />
```

Comment:
do we need separate component for that? idea was to put the table code here, otherwise this component is just a proxy